llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -985,6 +987,57 @@ getSizeFromDwarfHeaderAndSkip(BinaryStreamReader &Reader) {
   return Size;
 }
 
+/// Reads and returns the abbreviation offset field of a dwarf Compilation Unit
```

Comment:
Nit: we should pick one spelling for Dwarf and stick to it ;-)

Reply:
Sigh, yeah :(
For some reason spelling Dwarf is very mood dependent...